client/src/constants.tsx

```diff
@@ -8,3 +8,6 @@ export const EXPRESS_LOGIN_URL = '/login';
 export const REACT_LOGIN_URL = '/login';
```

Comment:
Since the constants will hold alot of vars, its best if we namespaced the vars with the appropriate prefix e.g
// URLS
URL_FOO = '/foo';
URL_BAR = '/bar';

// Buttons
BUTTON_FOO = 'foo';
BUTTON_BAR = 'bar';